easysocial/easysocial.php

```diff
@@ -9,11 +9,13 @@
 
 jimport('joomla.plugin.plugin');
 
+ES::import('site:/controllers/controller');
```

Comment:
Added this line to EasySocial api compatible with EasySocial 2.1.0 Beta 1 package